src/common/dao/joblog.go

```diff
@@ -0,0 +1,40 @@
+package dao
+
+import (
+	"github.com/goharbor/harbor/src/common/models"
+	"github.com/astaxie/beego/orm"
+	"time"
+)
+
+// CreateOrUpdateJobLog ...
+func CreateOrUpdateJobLog(log *models.JobLog) (int64, error) {
```

Comment:
Does it need to return count?

Reply:
not necessary. always return 1 for Job_uuid is a unique index in DB.
just a result from ORM InsertOrUpdate.